Dockerfile

```diff
@@ -1,7 +1,15 @@
-FROM renovate/ruby@sha256:1c25e2b8e76557412434dd4c6a1c6057aafeae4f26418d119cf388bca265b295
+# renovate: datasource=rubygems depName=cocoapods versioning=ruby
+ARG COCOAPODS_VERSION=1.9.2
+
+FROM renovate/buildpack:2-ruby@sha256:c83d691fae1ad0d4594c22607d594cad152bdfdf436998268bb57a9b4329a890
+
+ARG RUBY_VERSION=2.7.1
+RUN install-tool ruby
 
 ARG COCOAPODS_VERSION
+RUN install-gem cocoapods
```

Comment:
Shouldn't this use a specified version?

Reply:
no, otherwise we need to rebuild every version, because the file changed. I've using the same pattern in other images, which works fine